pml/tests/test_plm.py

```diff
@@ -175,11 +178,27 @@ def start_plm_service(host):
     assert status.stdout.strip() == "active", f"PLM service is inactive: {status.stdout}"
     return start_plm
 
-# def test_pml_version(pml_version):
-#     """Test that percona-mongolink version output is correct"""
-#     pattern = r"^v\d+\.\d+ [a-f0-9]{7} \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$"
-#
-#     assert re.match(pattern, pml_version.stderr)
+def get_git_commit():
+    url = f"https://api.github.com/repos/percona/percona-mongolink/commits/release-{version}"
+    git_commit = requests.get(url)
+
+    if git_commit.status_code == 200:
+        return git_commit.json()["sha"][:7]
+    else:
+        print(f"Unable to obtain git commit, failed with status code: {git_commit.status_code}")
+        return False
+
+def test_pml_version(host):
+    """Test that percona-mongolink version output is correct"""
+    result = pml_version(host)
+    lines = result.stderr.split("\n")
+    parsed_config = {line.split(":")[0]: line.split(":")[1].strip() for line in lines[0:-1]}
+    assert parsed_config['Version'] == f"release-{version}", parsed_config
+    assert parsed_config['Platform'] == get_git_commit(), parsed_config
+    assert parsed_config['GitCommit'], parsed_config
+    assert parsed_config['GitBranch'], parsed_config
+    assert parsed_config['BuildTime'], parsed_config
+    assert parsed_config['GoVersion'], parsed_config
```

Comment:
Your check doesn't match output, which should look like:
Version:   v0.5.0
Platform:  linux
GitCommit: 3c8b832
GitBranch: release-0.5.0
BuildTime: 2025-05-23_08:59_UTC
GoVersion: go1.24.2